src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -532,4 +547,79 @@ public static void AbortFailed(ILogger logger, Exception exception)
             }
         }
     }
+
+    internal class StreamTracker
```

Comment:
Move this to it's own file (The lightbulb/ctrl-. should have a way to do this for you, as long as you do it **after** moving the other types to be nested types, see below)